core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -429,6 +432,24 @@ RouteBuilder fallback(boolean isFallback) {
         return this;
     }
 
+    /**
+     * Adds a {@link Route} that is supposed to be excluded from the {@link Route} built by this
+     * {@link RouteBuilder}.
+     */
+    RouteBuilder exclude(Route excludedRoute) {
```

Comment:
How about adding a shortcut for a path pattern for the `exclude`?
It would be convenient for simple cases.

Reply:
Good idea! I guess it's enough to add `exclude(String pathPattern)` to `RouteBuilder` and `*BindingBuilder`s. What do you think?

Reply:
Sounds good. :-)